pokt-foundation/docs
# Update RPC endpoints

1. **Updated Endpoints Table**:
   - The endpoints table has been updated with the latest RPC URLs to ensure users have accurate and up-to-date information.

2. **Video Update**:
   - Replaced the previous video with a new instructional video to better guide users on how to update their endpoints.

3. **Loom Video Embed Support**:
   - Created a custom Hugo shortcode to support embedding Loom videos within the documentation, enhancing our video content capabilities.

4. **Directory Restructure**:
   - Removed old subdirectories that were no longer needed, keeping the repository clean and well-organized.
   - Created a new directory named `update-endpoints` to house all related documents and files for updating endpoints, ensuring a structured and easily navigable repository.

5. **Testing**:
   - Ensure that the new video embeds display correctly on all pages.
   - Verify that all links in the updated endpoints table are correct and functioning as expected.
